packages/governance/src/paramGovernance/governParam.js

```diff
@@ -5,15 +5,18 @@ import { Far } from '@agoric/marshal';
 import { makePromiseKit } from '@agoric/promise-kit';
 import { sameStructure } from '@agoric/same-structure';
 
-import { q } from '@agoric/assert';
 import {
   ChoiceMethod,
   QuorumRule,
   ElectionType,
   looksLikeQuestionSpec,
 } from '../question.js';
 
-const { details: X } = assert;
+const { details: X, quote: q } = assert;
+
+// The electorate that governs changes to the contract's parameters. It must
+// be declared in the governed contract.
+const CONTRACT_ELECTORATE = 'Electorate';
```

Comment:
To attach a comment to a declaration, I've had better luck with JSDoc syntax.
That way, vs-code provides it in intellisense or whatever.

```suggestion
/** The electorate that governs changes to the contract's parameters. It must
   be declared in the governed contract. */
const CONTRACT_ELECTORATE = 'Electorate';
```

Reply:
good idea.